testscommon/utils.go

```diff
@@ -28,3 +32,19 @@ func CreateMemUnit() storage.Storer {
 	unit, _ := storageunit.NewStorageUnit(cache, persist)
 	return unit
 }
+
+// ReadInitialAccounts returns the genesis accounts from a file
+func ReadInitialAccounts(filePath string) ([]genesis.InitialAccountHandler, error) {
```

Comment:
This is the func we should also use in main files for sovereign/normal chain binary

Reply:
Also, we need unit tests for this

Reply:
create a `runTypeComponentsHelper` file for this and added unit tests. 
This will contain also the common `CreateArgsRunTypeComponents` in the next PR